streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -869,6 +869,9 @@ public synchronized void start() throws IllegalStateException, StreamsException
     /**
      * Shutdown this {@code KafkaStreams} instance by signaling all the threads to stop, and then wait for them to join.
      * This will block until all threads have stopped.
+     *
```

Comment:
Nit: if you want a new paragraph, you need to insert `<p>` -- otherwise, JavaDocs will render a single paragraph.